lib/hooks/index.js

```diff
@@ -114,6 +118,16 @@ function activate(agent) {
     moduleHook.patch(instrumentation.module, agent, instrumentation.version);
   }
 
+  for (var hookName in toInstrument) {
+    var regex = new RegExp('node_modules' + path.sep + hookName + path.sep);
+    for (var file in require.cache) {
+      if (file.match(regex)) {
+        logger.warn(file + ' from ' + hookName + ' was loaded before trace agent.');
```

Comment:
A better wording would be something like '$hookName tracing might not work as $file is loaded before trace-agent was initialized'